README.rst

```diff
@@ -24,7 +24,7 @@ Why use ScanCode?
   and more both in **source code** and **binary** files.
 
 - ScanCode provides the **most accurate license detection engine** and does a
-  full comparison (aka. diff or red line) between a database of license texts
+  full comparison (also known as, different or red line comparison) between a database of license texts
```

Comment:
Thank you!  `diff` is word here as in https://en.wikipedia.org/wiki/Diff so we really cannot use different
As for red line, it could be improved and there is also a word for that https://www.merriam-webster.com/legal/redlining

Reply:
so,i'll just make different go back to diff. will that be okay?